assets/main.css

```diff
@@ -26,6 +26,25 @@
 	margin-top: 0;
 }
 
+.batch-picker .button-secondary {
+	background: none !important;
```

Comment:
Is there an alternative approach that doesn't require overwriting styles with `!important`?